Examples/Run/Reconstruction/Common/HoughExample.cpp

```diff
@@ -0,0 +1,214 @@
+
+  // The geometry, material and decoration
+  auto geometry = Geometry::build(vm, *detector);
+  auto tGeometry = geometry.first;
+  auto contextDecorators = geometry.second;
+  auto rnd =
+      std::make_shared<RandomNumbers>(Options::readRandomNumbersConfig(vm));
+
+  // Add the decorator to the sequencer
+  for (auto cdr : contextDecorators) {
+    sequencer.addContextDecorator(cdr);
+  }
+
+  // Setup the magnetic field
+  Options::setupMagneticFieldServices(vm, sequencer);
+  auto magneticField = Options::readMagneticField(vm);
+
+  // Read the sim hits
+  auto simHitReaderCfg = setupSimHitReading(vm, sequencer);
+  // Read the particles
+  auto particleReader = setupParticleReading(vm, sequencer);
+
+  // Run the sim hits smearing
+  auto digiCfg = setupDigitization(vm, sequencer, rnd, tGeometry,
+                                   simHitReaderCfg.outputSimHits);
+
+  // Run the particle selection
+  // The pre-selection will select truth particles satisfying provided criteria
+  // from all particles read in by particle reader for further processing. It
+  // has no impact on the truth hits read-in by the cluster reader.
+  TruthSeedSelector::Config particleSelectorCfg;
+  particleSelectorCfg.inputParticles = particleReader.outputParticles;
+  particleSelectorCfg.inputMeasurementParticlesMap =
+      digiCfg.outputMeasurementParticlesMap;
+  particleSelectorCfg.outputParticles = "particles_selected";
+  particleSelectorCfg.ptMin = 1_GeV;
+  particleSelectorCfg.etaMax = 2.5;
+  particleSelectorCfg.etaMin = -2.5;
+  particleSelectorCfg.nHitsMin = 9;
+  sequencer.addAlgorithm(
+      std::make_shared<TruthSeedSelector>(particleSelectorCfg, logLevel));
+
+  // The selected particles
+  const auto& inputParticles = particleSelectorCfg.outputParticles;
+
+  // Create space points
+  SpacePointMaker::Config spCfg = Options::readSpacePointMakerConfig(vm);
+  spCfg.inputSourceLinks = digiCfg.outputSourceLinks;
+  spCfg.inputMeasurements = digiCfg.outputMeasurements;
+  spCfg.outputSpacePoints = "spacepoints";
+  spCfg.trackingGeometry = tGeometry;
+  sequencer.addAlgorithm(std::make_shared<SpacePointMaker>(spCfg, logLevel));
+
+  // Hough algorithm
+  ActsExamples::HoughTransformSeeder::Config houghCfg;
+  houghCfg.inputSpacePoints = {
+      spCfg.outputSpacePoints,
+  };
+  houghCfg.outputSeeds = "seeds";
+  houghCfg.outputProtoTracks = "prototracks";
+
+  houghCfg.inputMeasurements = digiCfg.outputMeasurements;
+  houghCfg.trackingGeometry = tGeometry;
+  houghCfg.inputSourceLinks = digiCfg.outputSourceLinks;
+  houghCfg.geometrySelection = {Acts::GeometryIdentifier().setVolume(0)};
+
+  houghCfg.m_subRegions = {0, 1};
+  houghCfg.m_traceHits = true;
+
+  houghCfg.m_xMin = -0.05;  // minphi
+  houghCfg.m_xMax = 0.25;   // maxphi
+  houghCfg.m_yMin = -1.25;  // min q/pt, -1/0.8 GeV  for extension
+  houghCfg.m_yMax = 1.25;   // max q/pt, -1/0.8 GeV  for extension
+
+  houghCfg.m_imageSize_x = 216;
```

Comment:
the default is 7000, is this on purpose the same as m_imageSize_y? This would mean ~40000 bins. With these values I can believe the 70ms, with the 7000 phiBins it'd wager it would go out of memory or take *very* long.